man/nss-resolve.xml

```diff
@@ -66,7 +67,7 @@
 group:          compat mymachines systemd
 shadow:         compat
 
-hosts:          files mymachines <command>resolve [!UNAVAIL=return]</command> dns myhostname
+hosts:          mymachines <command>resolve [!UNAVAIL=return]</command> myhostname files dns
```

Comment:
```suggestion
hosts:          mymachines <literal>resolve [!UNAVAIL=return]</literal> myhostname files dns
```

Reply:
And here the opposite — `<command>` is used to highlight the change. (Yeah, it is a misuse, but docbook doesn't have the concept of non-semantic tags, so we need to take such shortcuts.) And `<literal>` would give us quotation marks, which we don't want.